src/index.ts

```diff
@@ -9,23 +9,23 @@ import {
   AudienceDefinition,
 } from '../types';
 
-interface Config {
-  pageFeatureGetters: PageFeatureGetter[];
+interface Config<T> {
+  pageFeatureGetters: PageFeatureGetter<T>[];
   audienceDefinitions: AudienceDefinition[];
   vendorIds?: number[];
   omitGdprConsent?: boolean;
 }
 
 // TODO: we need to give a way to consumers to ensure this does not
 // run multiple times on a single page load.
```

Comment:
I think we can remove this todo, as this is handled by the loader in the wrapper.

Do you think it is needed / possible in edgekit itself?

Reply:
I don't think it's needed
It's actually useful for the tests to be able to run it multiple times too